benchmark/cxx-source/CreateObjects.swift

```diff
@@ -0,0 +1,31 @@
+//===--- CreateObjects.swift ----------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+// This is a simple test that creates thousands of C++ objects and does nothing
+// with them.
+
+import TestsUtils
+import CxxTypes
+
+public let CreateObjects = BenchmarkInfo(
+  name: "CreateObjects",
+  runFunction: run_CreateObjects,
+  tags: [.validation, .bridging, .cpubench])
```

Comment:
> A benchmark should only be tagged "cpubench" after a full performance investigation of the benchmark has been completed to determine that it is a good representation of future Swift performance.

https://github.com/apple/swift/blob/de3af89539b9812f38a44730d7b468c2651eb10d/benchmark/utils/TestsUtils.swift#L64,L66

Reply:
Oh, my bad. I'll remove it.